submissions/kirill8210/dom-task/js/mobile.js

```diff
@@ -0,0 +1,17 @@
+const iconMenu = document.querySelector('.icon-menu');
+const menu = document.querySelector('.nav');
+const iconClose = document.querySelector('.icon-close');
+
+const addMenu = () =>{
+    menu.style.left = '0%';
+};
+
+const removeMenu = () =>{
+    menu.style.left = '-100%';
```

Comment:
Use a methods `Element.classList.add()/remove()/toggle()` of the element.

Reply:
ok